source/common/router/upstream_request.h

```diff
@@ -179,6 +180,8 @@ class UpstreamRequest : public Logger::Loggable<Logger::Id::router>,
   StreamInfo::StreamInfo& streamInfo() { return stream_info_; }
   bool hadUpstream() const { return had_upstream_; }
 
+  const Http::RequestHeaderMap& getRequestHeaders() const { return request_headers_; }
+
```

Comment:
And this is unnecessary for above reason.

Reply:
Removed.